finish_release.py

```diff
@@ -40,6 +44,39 @@ def tag_release(version):
     check_call(['git', 'push', '--follow-tags'])
 
 
+def set_release_date(version):
+    """Sets the release date(s) in RELEASE.rst for any versions missing it"""
+    print("Setting release date...")
+    release_filename = "RELEASE.rst"
+    if not os.path.isfile(release_filename):
+        return
+    date_format = "%Y-%m-%d"
```

Comment:
Can we get the date formatted as "July 12, 2018" -- I think that's "%b %d, %Y"

Reply:
✅